resources/pvc.py

```diff
@@ -0,0 +1,67 @@
+"""
+General PVC object
+"""
+import tempfile
+import logging
+
+from resources.ocs import OCS
+
+log = logging.getLogger(__name__)
+
+
+class PVC(OCS):
+    """
+    A basic PersistentVolumeClaim kind resource
+    """
+
+    def __init__(self, **kwargs):
+        """
+        Initializer function
+        kwargs:
+            See parent class for kwargs information
+        """
+        super(PVC, self).__init__(**kwargs)
+        self.temp_yaml = tempfile.NamedTemporaryFile(
+            mode='w+', prefix='PVC_', delete=False
+        )
+
+    def get_status(self):
```

Comment:
isn't it relevant for any other kind?

Reply:
I'm not sure if all kinds have status.phase as status